<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateVerificationCodeClass</class>
 <widget class="QMainWindow" name="GenerateVerificationCodeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>645</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>GenerateVerificationCode</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_GenerateCode">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>验证码生成</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_putstorage">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>入库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_del">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>注销</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_toppouring">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>上注</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>451</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_number_code">
    <property name="geometry">
     <rect>
      <x>218</x>
      <y>120</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>150</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>250</string>
     </property>
    </item>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>461</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_login">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_register">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_satellite_id">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>HS-1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HS-2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HS-3</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GenerateVerificationCode.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_putstorage</sender>
   <signal>clicked()</signal>
   <receiver>GenerateVerificationCodeClass</receiver>
   <slot>btn_get_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_get_clicked()</slot>
 </slots>
</ui>
